trino/trino_test.go

```diff
@@ -275,6 +275,52 @@ func TestQueryFailure(t *testing.T) {
 	}
 }
 
+func TestExec(t *testing.T) {
```

Comment:
I think this should go to `integration_test.go`

Reply:
Right, there's a few other tests in here that should be moved but I'll leave this to another PR.